src/components/pages/ParentBooking/ParentBookingCard.js

```diff
@@ -1,81 +1,343 @@
     size,
-    course_id,
+    min_age,
+    max_age,
   } = props.booking;
 
+  const { Content } = Layout;
   const { addToCart } = props;
+  const { Item } = Form;
+  const { button } = Select;
+  const { Title } = Typography;
 
   const { authState } = useOktaAuth();
   const { idToken } = authState;
+
   const handleClick = e => {
     axiosWithAuth(idToken)
-      .post(
-        '/children/1/enrollments', // TODO: Hook this request up to pass the ID of the parent/child involved once we have this data in state.
-        { child_id: 1, class_id: course_id, completed: true }
-      )
+      .post()
       .then(res => console.log(res)) // TODO: Let's perform some action with this result.
       .catch(err => console.log(`message: ${err.message}`));
   };
 
-  const data = [
-    { title: 'student name', text: child_name },
-    { title: 'course', text: subject },
-    { title: 'description', text: description },
-    { title: 'first day of class', text: dateConverter(start_date) },
-    { title: 'last day of class', text: dateConverter(end_date) },
-    {
-      title: 'time',
-      text: `${timeConverter(start_time)} - ${timeConverter(end_time)}`,
-    },
-    { title: 'location', text: location },
-    { title: 'instructor', text: instructor_name },
-    { title: 'class size', text: size },
-  ];
+  // const data = [
+  //   { title: 'student name', text: child_name },
+  //   { title: 'course', text: subject },
+  //   { title: 'description', text: description },
+  //   { title: 'first day of class', text: dateConverter(start_date) },
+  //   { title: 'last day of class', text: dateConverter(end_date) },
+  //   {
+  //     title: 'time',
+  //     text: `${timeConverter(start_time)} - ${timeConverter(end_time)}`,
+  //   },
+  //   { title: 'location', text: location },
+  //   { title: 'instructor', text: instructor_name },
+  //   { title: 'class size', text: size },
+  // ];
 
   const handleAddCourse = booking => {
     addToCart(booking);
   };
 
   return (
-    <Card title={subject} style={{ width: 280 }} hoverable="true">
-      {data.map((itm, idx) => {
-        return (
-          <div key={idx}>
-            {itm.title}: {itm.text}
+    <Layout className="il__top">
+      <Content className="il__top__topContent">
+        <div className="il__top__topContent__container">
+          <div className="il__top__topContent__main">
+            <Title
+              className="il__heading il__top__topContent__heading"
+              level={2}
+              style={{ color: '#3a072c', textAlign: 'center' }}
+            >
+              BOOK WITH US.
+              <br />
+              LEARN MORE THAN JUST CODE!
+            </Title>
+            <Form className="il__top__form" size={'large'} layout="inline">
+              <Input.Group
+                compact
+                // style={{ display: 'flex', flexDirection: 'column' }}
+              >
+                <div>
+                  <div
+                    style={{
```

Comment:
You might want to consider adding the styling in a separate CSS file to clean up the code a bit

Reply:
Yes absolutely we were just doing this to prevent switching between screens so much. This is still a draft PR at this point! Thank you!